Moto/E5Plus/AndroidManifest.xml

```diff
@@ -0,0 +1,10 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+        package="me.phh.treble.overlay.moto.e5plus"
+        android:versionCode="1"
+        android:versionName="1.0">
+        <overlay android:targetPackage="android"
+                android:requiredSystemPropertyName="ro.vendor.build.fingerprint"
+                android:requiredSystemPropertyValue="motorola/hannah_t/hannah:8.0.0/OCP27.91-150-8/2:user/release-keys"
```

Comment:
You're excluding any other vendor from working with this overlay. Follow these devices: https://github.com/phhusson/vendor_hardware_overlay/tree/master/Moto

Reply:
Do you mean, replace fingerprint with product name?

Reply:
Nevermind, edited it